### DISCUSSION TITLE
Velero operator installation

### DISCUSSION THREAD

**Discussion by nicolasdonato** [vmware-tanzu/velero#7249] [Community support Q&A]
Hello the Velero team,

I'm using Velero and it is a great backup application !!

I search a way for automated the installation of Velero into K8S cluster using the CRD operators, like the way you used with Backup and ScheduleBackup CRDs.

In this case, I would like to know if you have created an installation operator of Velero like https://github.com/openshift/managed-velero-operator and put this installation CRD into the hub named [OperatorHub](https://operatorhub.io).
I tried to put in place a cluster custom installation based on installation CRD operator like [Grafana](https://operatorhub.io/operator/grafana-operator).

![image](https://github.com/vmware-tanzu/velero/assets/9198793/74288fa0-2fd2-4cdc-a5c7-da9bd3c4c85d)

Do you have this capability to put an installation operator on OperatorHub ?

Thank you in advance.

**Comment by shubham-pampattiwar** [comment]
Hey @nicolasdonato you can try using OADP Operator: https://github.com/openshift/oadp-operator

**Reply by kaovilai** [reply]
+1, oadp-operator repo should be easier for you to port to operatorhub for k8s.

**Comment by streammaster87** [comment]
Setting up the Velero operator for streamlined installation is akin to optimizing a streaming app for seamless performance. Both processes aim to enhance efficiency and provide users with a smoother experience. Just as Velero ensures reliable backups for Kubernetes, a well-optimized [live streaming](https://hotstarmodapk.net/) app guarantees uninterrupted entertainment. Dive into the installation process to elevate the functionality and reliability of your streaming platform.
